OpenRA.Mods.Common/Widgets/ProductionTabsWidget.cs

```diff
@@ -35,7 +35,7 @@ public class ProductionTabGroup
 
 		public void Update(IEnumerable<ProductionQueue> allQueues)
 		{
-			var queues = allQueues.Where(q => q.Info.Group == Group).ToList();
+			var queues = allQueues.Where(q => q.Info.Group == Group && q.Enabled).ToList();
```

Comment:
I know this is very minor and there's much bigger issues with the surrounding code, but can we 
```suggestion
			var queues = allQueues.Where(q => q.Enabled && q.Info.Group == Group).ToList();
```